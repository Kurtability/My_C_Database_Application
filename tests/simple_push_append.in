push a 1
set a 1 2 3
push a 4 5
get a
append a 4 5
get a
set a 3
push a 2 1
get a
append a 4 5
get a
bye
