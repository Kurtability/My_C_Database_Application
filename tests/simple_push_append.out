> push a 1 
no such key

> set a 1 2 3 
ok

> push a 4 5
ok

> get a 
[5 4 1 2 3]

> append a 4 5
ok

> get a 
[5 4 1 2 3 4 5]

> set a 3
ok

> push a 2 1
ok

> get a 
[1 2 3]

> append a 4 5
ok

> get a 
[1 2 3 4 5]

> bye
bye
